includes/wcs-switch-functions.php

```diff
@@ -25,18 +25,11 @@ function wcs_order_contains_switch( $order ) {
 	}
 
 	if ( ! wcs_is_order( $order ) || wcs_order_contains_renewal( $order ) ) {
```

Comment:
It seems really strange that we're checking if an order contains a renewal. 🤔 

I can't think of a reason why we'd be doing that. I can understand that renewals and switch orders are mutually exclusive, but enforcing that in this way seems odd.

Reply:
I went back to the original function and it looks like this check for whether the order contains a renewal is part of the original function, however, it was more performant because it was recorded on the order object if it was a renewal. See the original code here: https://github.com/Automattic/woocommerce-subscriptions-core/commit/bfc96202b557b7d7e39cea4478330c1bddde9c6f

With that in mind, I think we can get rid of it.

Reply:
Funny how ancient code like this has survived all this time. I've updated this function in https://github.com/Automattic/woocommerce-subscriptions-core/commit/ad40418779131969ca359859e2dce84592e317f8